### PR TITLE
feat: add onScrollerCreated callback

### PR DESCRIPTION
**Pull request by gcedo**

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -76,6 +76,40 @@ describe('Carousel', () => {
...
+          visibleItems={4}
+        >
+          {horizontalNodes}
+        </Carousel>
+      );
+      spiedOnScrollerCreated.should.have.been.called.exactly(1);
+      spiedOnScrollerCreated.should.have.been.called.with(rendered.get(0).scroller);
+    });
+
+    it('renders when there is no function', () => {
```
Comment (fabiosantoscode) [inline test/index.js]: Nth: This case is already implicitly checked in all other tests

**Comment by fabiosantoscode** [comment]
Single Nth, but LGTM, feel free to merge.

File: test/index.js (hunk 1 above)
Comment (gcedo) [inline test/index.js]: True, I thought that making it explicit would have been better, but the other cases would fail anyway...

**Comment by fabiosantoscode** [comment]
:rainbow:

**Pull request merged**